.ci/scripts/prepare_slack_data.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+"""
+This script is designed to be used in a GitHub Actions workflow to send Slack notifications.
+It reads environment variables set by the GitHub Actions runtime, validates them, and
+constructs a Slack payload based on the workflow status.
+"""
+import os
+import json
+
+
+def check_env_var(env_var: str) -> str:
+    """
+    Retrieve the value of the specified environment variable.
+
+    Parameters:
+        env_var (str): The name of the environment variable to retrieve.
+
+    Returns:
+        str: The value of the specified environment variable.
+    """
+    value = os.environ.get(env_var)
+    if value is None or value == "":
```

Comment:
would that be the same?
```suggestion
    if not value
```

Reply:
fixed